taichi/com_mgr/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# ./taichi/com_mgr/CMakeLists.txt
+
+add_library(compilation_manager)
```

Comment:
nit: can we name the folder `compilation_manager` or something better? `com` isn't very informative here :P

Reply:
OK